API Blueprint Specification.md

```diff
@@ -478,6 +488,189 @@ Name and description of the API
+              "type": "error"
+            }
+
+#### Request Authentication section
+Description of the request level authentication settings.
+
+If defined inside a [Request section](#def-request-section) of a HTTP transaction example, it implies that the API's response will be the same as the respective response when a valid authentication is sent along with the above mentioned request. This allows another HTTP transaction example to be specified for the cases where an authentication is not needed.
+
+This section **should** not have any nested sections.
+
+If a list of scheme names is provided after the `Authentication` keyword, then only those particular authentication schemes will be allowed for the respective HTTP transaction.
+
+```
++ Authentication
+```
+
+```
++ Authentication: basic, token
+```
+
+##### Example
+
+    ## GET /users/bond
+
+    + Response 200 (application/json)
+
+            {
+                "username": "bond"
+            }
+
+    + Request
+        + Authentication
+
+    + Response 200 (application/json)
+
+            {
+                "username": "bond",
+                "email": "james@bo.nd",
+                "bio": "I am 007"
+            }
+
+#### Action Authentication section
+Description of the action level authentication settings.
+
+If defined, all the [Request sections](#def-requestion-section) of the respective [Action section](#def-action-section) inherits the authentication settings unless specified otherwise.
+
+This section **should** not have any nested sections.
+
+##### Example
+
+    ## Retrieve User [GET]
+
+    + Authentication
+
+    + Response 200
+
+            { ... }
+
+---
+
+<a name="def-authenticationschemes-section"></a>
+## Authentication schemes section
+- **Parent sections:** [Authentication section](#def-authentication-section)
+- **Nested sections:** See below
+- **Markdown entity:** list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication Schemes` keyword.
+
+    + Authentication Schemes
+
+#### Description
+Description of the list of authentication schemes supported by the API.
+
+This section **must** be composed of nested list items only. This section **must not** contain any other elements. Each list item describes a single Authentication scheme.
+
+    + <scheme name> (<base scheme type>) - <description>
+
+        + `<key 1>`: <value 1>
+        + `<key 2>`: <value 2>
+        ...
+        + `<key N>`: <value N>
+
+Where:
+
++ `<scheme name>` is the scheme name as written in [Authentication section](#def-authentication-section) under an [Action section](#def-action-section).
++ `<description>` is any **optional** Markdown-formatted description of the authentication scheme.
++ `<base scheme type>` is the authentication scheme type as expected by the API (e.g. "Basic Auth Scheme"). See below for all the available base authentication schemes.
++ `<key n>` represents a key of an customizable option in the base authentication scheme.
```

Comment:
you have just introduced "customizable options" – what are they where were they defined? I believe these terms should be mentioned early in the description above and this breakdown should just "reference" the term